README.md

```diff
@@ -1,4 +1,67 @@
-# **Brain 4CE** - An EEG-based Brain-Computer Interface
-> *Where thoughts become motion*.
-## Designed by BU ECE 2023 Senior Design Capstone Team 4: 
-**Alex Johnson, Dayanna De La Torres, Jonathan Mikalov, Brendan Shortall, & Mitchell Gilmore**
\ No newline at end of file
+<div style="text-align: center;">
+
+# **Brain-4ce** - An EEG-based Brain-Computer Interface
+
+</div>
+
+
+<p align="center">
+  <img src="./Documents/brain_4ce_logo.png" alt="Brain 4ce Logo" style="display:block; margin:auto;" width="350">
```

Comment:
Can we decrease the width of the logo to 100-200px?

Reply:
The logo becomes too small and illegible at <=200px